### DISCUSSION TITLE
Include flightId in GET /dsp/orders/

### DISCUSSION THREAD

**Discussion by bartektrybala** [amzn/ads-advanced-tools-docs#62] [Ideas]
Hi, to make an update on order you need _flightId_ field. Creation response returns only _amz_id_, so to do an update I have to call **GET** _/dsp/order/{orderId}_ once again to retrieve _flightId_.

It would be much easier if batch GET orders API was returning the _flight_id_. You can retrieve _flightId_ in a batch requests and update all of your orders (in a sync script for example). Is it possible to make it?

Maybe someone else got the same annoying issue? I'm happy to hear it.

**Comment by jakub-borusewicz** [comment]
Yeah, that would be convenient

**Comment by elizafarley** [comment]
Thanks for your feedback! We are working on a new version of the DSP campaign management APIs, and planning to include this functionality. We'll post in the [release notes](https://advertising.amazon.com/API/docs/en-us/info/release-notes) (and here!) if this becomes available.

**Reply by bartektrybala** [reply]
Great to hear! I’m looking forward to it

**Comment by AdsAPIPM** [comment]
Would it be easier if you didnt need the flight id to make an update?

**Reply by bartektrybala** [reply]
You have to identify them in a way

**Reply by AdsAPIPM** [reply]
You could identify them by the dates

**Reply by davidmoss** [reply]
Yes that would be easier
